python/ray/air/checkpoint.py

```diff
@@ -351,6 +351,15 @@ def from_directory(cls, path: str) -> "Checkpoint":
         """
         return cls(local_path=path)
 
+    @classmethod
+    def _from_checkpoint(cls, other: "Checkpoint") -> "Checkpoint":
```

Comment:
Should we port the others to use this one?

Reply:
I will